extensions/ql-vscode/src/stories/Overview.stories.mdx

```diff
@@ -45,12 +45,12 @@ select the **Dark+** theme. You can use **Preferences: Color Theme** in the *Com
 4. Select **Developer: Open WebView Developer Tools**
 5. Now, you will need to find the `<html>` element in the lowest-level `<iframe>`. See the image below:
 
-<img src={iframeImage} />
+   <img src={iframeImage} />
```

Comment:
This indentation makes the images part of the list, and prevents markdown list saying:

```
MD029/ol-prefix Ordered list item prefix [Expected: 1; Actual: 6; Style: 1/1/1] https://github.com/DavidAnson/markdownlint/blob/v0.27.0/doc/md029.md
```